pages/index.js

```diff
@@ -24,20 +24,30 @@ import GitHubAccelerator from "@components/GitHubAccelerator";
 import Alert from "@components/Alert";
 import config from "@config/app.json";
 import Newsletter from "@components/Newsletter";
+import UserProfile from "@components/user/UserProfile";
+
 
 export async function getStaticProps() {
   const pageConfig = config.isr.homepage; // Fetch the specific configuration for this page
 
   const { stats: totalStats } = await getTotalStats();
   const { stats: todayStats } = await getTodayStats();
 
+  // Fetch random Profiles
+  const response = await fetch(`${process.env.NEXT_PUBLIC_BASE_URL}/api/discover/random`);
```

Comment:
We do not use `process` but `zod` please check other files for example usage

Reply:
Done!